bouncer/blacklist/views.py

```diff
@@ -27,3 +31,21 @@ def get(self, request):
             response = {"kind": "ip", "value": ip, "result": result}
             result_list.append(response)
         return Response(result_list)
+
+
+class PostRequestViewSet(viewsets.ViewSet):
```

Comment:
This can be merged with the view above, since they refer to different `request.method`s